### PR TITLE
Shard related code removed

### PR DESCRIPTION
**Pull request by motoras**
For details see [ticket](https://radixdlt.atlassian.net/browse/RPNV1-309)

### CODE REVIEW

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 1]
```diff
@@ -59,26 +60,26 @@
 
 	public static Atom create(ParticleGroup particleGroup, long timestamp) {
 		return new Atom(
-			ImmutableList.of(particleGroup),
-			ImmutableMap.of(METADATA_TIMESTAMP_KEY, String.valueOf(timestamp)),
-			ImmutableMap.of()
-		);
+				ImmutableList.of(particleGroup),
```
Comment (talekhinezh) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Some funky tabs

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 2]
```diff
@@ -222,7 +201,7 @@ public Hash getHash() {
 	}
 
 	public AID getAid() {
-		return AID.from(getHash(), getShards());
+		return AID.from(getHash().toByteArray());
```
Comment (talekhinezh) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java:203]: Like mentioned in other library, this change I think would require some synchronization with the javascript library

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 3]
```diff
@@ -143,16 +143,16 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
 					atom.spunParticles().forEach(s -> {
 						Map<Spin, Set<Atom>> spinParticleIndex = particleIndex.getOrDefault(s.getParticle(), Collections.emptyMap());
 						spinParticleIndex.getOrDefault(s.getSpin(), Collections.emptySet())
-							.forEach(a -> {
```
Comment (talekhinezh) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Why the removal of tabs here?

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 4]
```diff
@@ -59,26 +60,26 @@
...
 	public static Atom create(ParticleGroup particleGroup, long timestamp) {
 		return new Atom(
-			ImmutableList.of(particleGroup),
-			ImmutableMap.of(METADATA_TIMESTAMP_KEY, String.valueOf(timestamp)),
-			ImmutableMap.of()
-		);
+				ImmutableList.of(particleGroup),
+				ImmutableMap.of(METADATA_TIMESTAMP_KEY, String.valueOf(timestamp)),
+				ImmutableMap.of()
+				);
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Most of these edits here look like a reformat without any other changes.  Please revert.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 5]
```diff
@@ -40,6 +40,7 @@
 import com.radixdlt.serialization.SerializerId2;
 
 import com.radixdlt.identifiers.RadixAddress;
+
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 6]
```diff
@@ -59,26 +60,26 @@
...
+				);
 	}
 
 	public static Atom create(List<ParticleGroup> particleGroups, long timestamp) {
 		return new Atom(
-			ImmutableList.copyOf(particleGroups),
-			ImmutableMap.of(METADATA_TIMESTAMP_KEY, String.valueOf(timestamp)),
-			ImmutableMap.of()
-		);
+				ImmutableList.copyOf(particleGroups),
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 7]
```diff
@@ -59,26 +60,26 @@
...
+				);
 	}
 
 	public static Atom create(List<ParticleGroup> particleGroups, Map<String, String> metaData) {
 		return new Atom(
-			ImmutableList.copyOf(particleGroups),
-			ImmutableMap.copyOf(metaData),
-			ImmutableMap.of()
-		);
+				ImmutableList.copyOf(particleGroups),
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 8]
```diff
@@ -110,10 +111,10 @@ private Atom() {
 	}
 
 	private Atom(
-		ImmutableList<ParticleGroup> particleGroups,
-		ImmutableMap<String, String> metaData,
-		ImmutableMap<String, ECDSASignature> signatures
-	) {
+			ImmutableList<ParticleGroup> particleGroups,
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 9]
```diff
@@ -134,34 +135,12 @@ public Atom addSignature(EUID signatureId, ECDSASignature signature) {
...
-	}
-
-	private Set<Long> getShards() {
-		return this.spunParticles()
-			.map(SpunParticle<Particle>::getParticle)
-			.map(Particle::getDestinations)
-			.flatMap(Set::stream)
-			.map(EUID::getShard)
-			.collect(Collectors.toSet());
+				this.particleGroups,
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java [hunk 10]
```diff
@@ -177,10 +156,10 @@ public Atom addSignature(EUID signatureId, ECDSASignature signature) {
 
 	public Stream<RadixAddress> addresses() {
 		return this.spunParticles()
-			.map(SpunParticle<Particle>::getParticle)
-			.map(Particle::getShardables)
-			.flatMap(Set::stream)
-			.distinct();
+				.map(SpunParticle<Particle>::getParticle)
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 2 above)
Comment (clstrfsck) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java:203]: Right, this is planned.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 11]
```diff
@@ -66,23 +66,23 @@
...
 		atom.particles(Spin.UP)
-			.forEach(p -> {
-				Map<Spin, Set<Atom>> particleSpinIndex = particleIndex.get(p);
-				particleSpinIndex.getOrDefault(Spin.DOWN, Collections.emptySet())
-					.forEach(a -> {
-						AtomObservation observation = atoms.get(a);
-						if (observation.getAtom().equals(atom)) {
-							return;
-						}
+		.forEach(p -> {
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 12]
```diff
@@ -247,32 +247,32 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
...
-						return false;
-					}
-					final Map<Spin, Set<Atom>> spinParticleIndex = e.getValue();
-					final boolean hasDown = spinParticleIndex.getOrDefault(Spin.DOWN, Collections.emptySet())
-						.stream().anyMatch(a -> atoms.get(a).isStore());
-					if (hasDown) {
-						return false;
-					}
+					.filter(e -> {
+						//					if (!e.getKey().getShardables().contains(address)) {
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Commented out code should be removed.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/actions/FetchAtomsRequestAction.java [hunk 13]
```diff
@@ -61,10 +59,10 @@ public RadixAddress getAddress() {
 		return address;
 	}
 
-	@Override
-	public Set<Long> getShards() {
-		return Collections.singleton(address.euid().getShard());
-	}
+	//	@Override
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/actions/FetchAtomsRequestAction.java]: Commented out code should be removed.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 14]
```diff
@@ -194,31 +181,31 @@ public void when_first_node_fails__then_second_node_should_connect() {
...
+				badPeer, mockedNodeState(WebSocketStatus.DISCONNECTED),
+				goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
+				)));
 
 		FindANodeEpic findANodeEpic = new FindANodeEpic(new GetFirstSelector());
 		TestObserver<RadixNodeAction> testObserver = TestObserver.create();
 
 		FindANodeRequestAction request = mock(FindANodeRequestAction.class);
-		when(request.getShards()).thenReturn(Collections.singleton(1L));
+
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java:192]: Extra blank line.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 15]
```diff
@@ -166,16 +166,16 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
 							particleIndex.put(s.getParticle(), spinParticleIndex);
 						}
 						spinParticleIndex.merge(
-							s.getSpin(),
-							Collections.singleton(atom),
-							(a, b) -> new ImmutableSet.Builder<Atom>().addAll(a).addAll(b).build()
-						);
+								s.getSpin(),
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 16]
```diff
@@ -166,16 +166,16 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
...
+								Collections.singleton(atom),
+								(a, b) -> new ImmutableSet.Builder<Atom>().addAll(a).addAll(b).build()
+								);
 					});
 				} else {
 					// Soft observation should not be able to update a hard state
 					// Only update if type changes
 					include = (!nextUpdate.isSoft() || lastUpdate.isSoft())
-						&& nextUpdate.getType() != lastUpdate.getType();
+							&& nextUpdate.getType() != lastUpdate.getType();
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 17]
```diff
@@ -236,9 +236,9 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
 	public Stream<Atom> getStoredAtoms(RadixAddress address) {
 		synchronized (lock) {
 			return atoms.entrySet().stream()
-				.filter(e -> e.getValue().isStore() && e.getKey().addresses().anyMatch(address::equals))
-				.map(Map.Entry::getValue)
-				.map(AtomObservation::getAtom);
+					.filter(e -> e.getValue().isStore() && e.getKey().addresses().anyMatch(address::equals))
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 18]
```diff
@@ -247,32 +247,32 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
...
+						final boolean hasDown = spinParticleIndex.getOrDefault(Spin.DOWN, Collections.emptySet())
+								.stream().anyMatch(a -> atoms.get(a).isStore());
+						if (hasDown) {
+							return false;
+						}
 
-					if (stagedUuid != null && stagedParticleIndex.getOrDefault(stagedUuid, Collections.emptyMap()).get(e.getKey()) == Spin.DOWN) {
-						return false;
-					}
+						if (stagedUuid != null && stagedParticleIndex.getOrDefault(stagedUuid, Collections.emptyMap()).get(e.getKey()) == Spin.DOWN) {
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 19]
```diff
@@ -247,32 +247,32 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
...
+							return false;
+						}
 
-					Set<Atom> uppingAtoms = spinParticleIndex.getOrDefault(Spin.UP, Collections.emptySet());
-					return uppingAtoms.stream().anyMatch(a -> atoms.get(a).isStore());
-				})
-				.map(Map.Entry::getKey)
-				.collect(Collectors.toSet());
+						Set<Atom> uppingAtoms = spinParticleIndex.getOrDefault(Spin.UP, Collections.emptySet());
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 20]
```diff
@@ -247,32 +247,32 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
...
+					})
+					.map(Map.Entry::getKey)
+					.collect(Collectors.toSet());
 
 			if (stagedUuid != null) {
 				stagedParticleIndex.getOrDefault(stagedUuid, Collections.emptyMap()).entrySet().stream()
-					.filter(e -> e.getValue() == Spin.UP)
-					.map(Entry::getKey)
-					.forEach(upParticles::add);
+				.filter(e -> e.getValue() == Spin.UP)
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java [hunk 21]
```diff
@@ -293,9 +293,9 @@ public void store(RadixAddress address, AtomObservation atomObservation) {
 				}
 				observers.add(emitter);
 				atoms.entrySet().stream()
-					.filter(e -> e.getValue().isStore() && e.getKey().addresses().anyMatch(address::equals))
-					.map(Map.Entry::getValue)
-					.forEach(emitter::onNext);
+				.filter(e -> e.getValue().isStore() && e.getKey().addresses().anyMatch(address::equals))
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNodeState.java [hunk 22]
```diff
@@ -41,7 +40,7 @@
 	private final RadixUniverseConfig universeConfig;
 
 	public RadixNodeState(RadixNode node, WebSocketStatus status, NodeRunnerData data, Integer version,
-	                      RadixUniverseConfig universeConfig) {
+			RadixUniverseConfig universeConfig) {
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNodeState.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/actions/FetchAtomsRequestAction.java [hunk 23]
```diff
@@ -37,9 +35,9 @@
 	private final RadixAddress address;
 
 	private FetchAtomsRequestAction(
-		String uuid,
-		RadixAddress address
-	) {
+			String uuid,
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/actions/FetchAtomsRequestAction.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java [hunk 24]
```diff
@@ -61,15 +57,15 @@ public FindANodeEpic(RadixPeerSelector selector) {
...
+	private List<RadixNodeAction> nextConnectionRequest(RadixNetworkState state) {
 		final Map<WebSocketStatus, List<RadixNode>> statusMap = Arrays.stream(WebSocketStatus.values())
-			.collect(Collectors.toMap(
-				Function.identity(),
-				s -> state.getNodeStates().entrySet().stream()
-					.filter(e -> e.getValue().getStatus().equals(s))
-					.map(Entry::getKey)
-					.collect(Collectors.toList())
-			));
+				.collect(Collectors.toMap(
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java [hunk 25]
```diff
@@ -79,84 +75,65 @@ public FindANodeEpic(RadixPeerSelector selector) {
...
 	}
 
-	private static List<RadixNode> getConnectedNodes(Set<Long> shards, RadixNetworkState state) {
+	private static List<RadixNode> getConnectedNodes(RadixNetworkState state) {
 		return state.getNodeStates().entrySet().stream()
-			.filter(entry -> entry.getValue().getStatus().equals(WebSocketStatus.CONNECTED))
-			.filter(entry -> entry.getValue().getShards().map(s -> s.intersects(shards)).orElse(false))
-			.map(Map.Entry::getKey)
-			.collect(Collectors.toList());
+				.filter(entry -> entry.getValue().getStatus().equals(WebSocketStatus.CONNECTED))
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java]: Unwanted whitespace changes.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java [hunk 26]
```diff
@@ -79,84 +75,65 @@ public FindANodeEpic(RadixPeerSelector selector) {
...
-						final RadixNode node = c.getNode();
-						return selectedNode
-							.map(RadixNodeAction::getNode)
-							.filter(selected -> !node.equals(selected))
-							.map(i -> CloseWebSocketAction.of(node));
-					});
-
-				return findConnectionActionsStream.concatWith(selectedNode).mergeWith(cleanupConnections);
-			});
+				.flatMap(a -> {
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/network/epics/FindANodeEpic.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 27]
```diff
@@ -91,16 +82,14 @@ public void disconnectedNodeConnectionTest() {
...
 
 		FindANodeRequestAction request = mock(FindANodeRequestAction.class);
-		when(request.getShards()).thenReturn(Collections.singleton(1L));
 
 		TestObserver<RadixNodeAction> testObserver = TestObserver.create();
 		findANodeEpic.epic(
-			Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
-			Observable.just(RadixNetworkState.of(node, nodeState)).concatWith(Observable.never())
-		)
+				Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 28]
```diff
@@ -114,26 +103,25 @@ public void dontConnectToAllNodesTest() {
 		RadixNode connectedPeer = mock(RadixNode.class);
 
 		Map<RadixNode, RadixNodeState> networkStateMap = IntStream.range(0, 100).boxed().collect(Collectors.toMap(
-			i -> {
-				if (i == 0) {
-					return connectedPeer;
-				}
+				i -> {
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 29]
```diff
@@ -114,26 +103,25 @@ public void dontConnectToAllNodesTest() {
...
+					if (i == 0) {
+						return connectedPeer;
+					}
 
-				RadixNode peer = mock(RadixNode.class);
-				return peer;
-			},
-			i -> mockedNodeState(i == 0 ? WebSocketStatus.CONNECTED : WebSocketStatus.DISCONNECTED)
-		));
+					RadixNode peer = mock(RadixNode.class);
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 30]
```diff
@@ -114,26 +103,25 @@ public void dontConnectToAllNodesTest() {
...
 		FindANodeRequestAction request = mock(FindANodeRequestAction.class);
-		when(request.getShards()).thenReturn(Collections.singleton(1L));
 
 		FindANodeEpic findANodeFunction = new FindANodeEpic(new GetFirstSelector());
 		TestObserver<RadixNodeAction> testObserver = TestObserver.create();
 		findANodeFunction.epic(
-			Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
-			Observable.just(new RadixNetworkState(networkStateMap)).concatWith(Observable.never())
-		)
+				Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 31]
```diff
@@ -149,34 +137,33 @@ public void when_first_node_takes_too_long__then_second_node_should_connect() {
 
 
 		networkState.onNext(new RadixNetworkState(ImmutableMap.of(
-			badPeer, mockedNodeState(WebSocketStatus.DISCONNECTED),
-			goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
-		)));
+				badPeer, mockedNodeState(WebSocketStatus.DISCONNECTED),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 32]
```diff
@@ -149,34 +137,33 @@ public void when_first_node_takes_too_long__then_second_node_should_connect() {
...
-		when(request.getShards()).thenReturn(Collections.singleton(1L));
 
 		findANodeEpic.epic(
-			Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
-			networkState
-		)
-			.doOnNext(i -> {
-				if (i.getNode().equals(badPeer)) {
-					networkState.onNext(new RadixNetworkState(ImmutableMap.of(
+				Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 33]
```diff
@@ -149,34 +137,33 @@ public void when_first_node_takes_too_long__then_second_node_should_connect() {
...
+				)
+		.doOnNext(i -> {
+			if (i.getNode().equals(badPeer)) {
+				networkState.onNext(new RadixNetworkState(ImmutableMap.of(
 						badPeer, mockedNodeState(WebSocketStatus.CONNECTING),
 						goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
-					)));
-				} else {
-					networkState.onNext(new RadixNetworkState(ImmutableMap.of(
+						)));
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 34]
```diff
@@ -149,34 +137,33 @@ public void when_first_node_takes_too_long__then_second_node_should_connect() {
...
+			} else {
+				networkState.onNext(new RadixNetworkState(ImmutableMap.of(
 						badPeer, mockedNodeState(WebSocketStatus.CONNECTING),
 						goodPeer, mockedNodeState(WebSocketStatus.CONNECTED)
-					)));
-				}
-			})
-			.subscribe(testObserver);
+						)));
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 35]
```diff
@@ -194,31 +181,31 @@ public void when_first_node_fails__then_second_node_should_connect() {
 
 
 		networkState.onNext(new RadixNetworkState(ImmutableMap.of(
-			badPeer, mockedNodeState(WebSocketStatus.DISCONNECTED),
-			goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
-		)));
+				badPeer, mockedNodeState(WebSocketStatus.DISCONNECTED),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 36]
```diff
@@ -194,31 +181,31 @@ public void when_first_node_fails__then_second_node_should_connect() {
...
 
 		FindANodeRequestAction request = mock(FindANodeRequestAction.class);
-		when(request.getShards()).thenReturn(Collections.singleton(1L));
+
 
 		findANodeEpic.epic(
-			Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
-			networkState
-		)
+				Observable.<RadixNodeAction>just(request).concatWith(Observable.never()),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 37]
```diff
@@ -194,31 +181,31 @@ public void when_first_node_fails__then_second_node_should_connect() {
...
+				networkState
+				)
 		.doOnNext(i -> {
 			if (i.getNode().equals(badPeer)) {
 				networkState.onNext(new RadixNetworkState(ImmutableMap.of(
-					badPeer, mockedNodeState(WebSocketStatus.FAILED),
-					goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
-				)));
+						badPeer, mockedNodeState(WebSocketStatus.FAILED),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java [hunk 38]
```diff
@@ -194,31 +181,31 @@ public void when_first_node_fails__then_second_node_should_connect() {
...
+						goodPeer, mockedNodeState(WebSocketStatus.DISCONNECTED)
+						)));
 			} else {
 				networkState.onNext(new RadixNetworkState(ImmutableMap.of(
-					badPeer, mockedNodeState(WebSocketStatus.FAILED),
-					goodPeer, mockedNodeState(WebSocketStatus.CONNECTED)
-				)));
+						badPeer, mockedNodeState(WebSocketStatus.FAILED),
```
Comment (clstrfsck) [inline radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Unwanted whitespace changes.

**Review by clstrfsck — CHANGES_REQUESTED** [review]
Mostly good, but need to turn off automatic code reformatting on save.

Please fix up the unnecessary whitespace changes and remove commented out code.

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 4 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 6 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 8 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java (hunk 34 above)
Comment (motoras) [reply radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Fixed

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java (hunk 36 above)
Comment (motoras) [reply radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Fixed

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java (hunk 38 above)
Comment (motoras) [reply radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 1 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java (hunk 3 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 7 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java (hunk 37 above)
Comment (motoras) [reply radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Fixed

File: radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java (hunk 35 above)
Comment (motoras) [reply radixdlt-java/src/test/java/com/radixdlt/client/core/network/epics/FindANodeEpicTest.java]: Fixed

File: radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java (hunk 5 above)
Comment (motoras) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java]: Fixed

**Review by clstrfsck — APPROVED** [review]

**Pull request merged**